README.md

```diff
@@ -107,11 +108,42 @@ the file in `vars/gpg-keys`.
 ### Logging into ArgoCD
 
 ```bash
-# Login to ArgoCD
 ARGOCD_ROUTE=$(oc get route argocd-server -o jsonpath='{.spec.host}')
+# Login to ArgoCD as admin user
 argocd --insecure --grpc-web login ${ARGOCD_ROUTE}:443 --username admin --password ${ARGOCD_SERVER_PASSWORD}
+# Login to ArgoCD via SSO
+argocd --insecure --grpc-web login ${ARGOCD_ROUTE}:443 --sso
+```
+
+## Adding Clusters and namespaces
+
+```bash
+argocd cluster add <cluster_context> --namespace <application_namespace>
 ```
 
+Note: You can find your cluster-contexts by going into your kubeconfig or by running argocd cluster add with no arguments.
```

Comment:
Maybe also mention that they can also just loginto the cluster, switch to the namespace they want to deploy to and run `kubectl config current-context`